doc/security/specs/secure_boot/index.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "Secure Boot"
+---
+
+<p style="text-align: right">
+Contributors(s):
+  <a href="https://github.com/moidx">Miguel Osorio</a>,
+  <a href="https://github.com/moidx">Garret Kelly</a>
+</p>
+
+<p style="color: red; text-align: right;">
+  Status: Pre-RFC
+</p>
+
+# Overview
+
+The purpose of secure boot is to ensure that only payloads which have been
+verified are executed as the system boots. This document describes the process
+by which OpenTitan aims to ensure this property holds, as well as the services
+that the secure boot chain provides to software, such as attestation and
+firmware upgrade support.
+
+# References
+
+*   [Attestation][attestation]
+*   [Ownership Transfer][ownership-transfer]
+*   [OpenTitan Secure Boot HW Support][secure-boot-hw-support]
+*   [OpenTitan Flash][ot-flash]
+
+# Terminology
+
+## RISC-V Concepts:
+
+*   **PMP:** Physical Memory Protection unit from the
```

Comment:
nit: Add note on ePMP being under consideration and potential changes to the spec.

Reply:
Done.